README.md

```diff
@@ -1,30 +1,177 @@
+    // can go over the internet.
+    fetch('/api/fruit=?eaten=true').then((data) => {
+      this.setState({ fruitList: data });
+    });
+  }
+
+  render() {
+    return (
+      <ul>
+        {this.state.fruitList.map((fruit) => {
+          return <li key={fruit}>{fruit}</li>;
+        })}
+      </ul>
+    ); 
+  }
+}
+```
+
+What we have here is a component that fetches a list of already eaten fruit and
+then renders that to a list. So what's the problem? Looks fine, right?
+
+Well...in this simple case everything looks good. But remember, we've only implemented one of
+our components. The `FruitNotEaten` component will also have to be able to fetch
+and update its state. Already, we can see that we're starting to repeat ourselves
+unnecessarily. The fetching process for fruit isn't really all that different in 
+each component. So why do we need to implement it twice? Plus, in general, we know
+that we should minimize state as much as possible to avoid bugs, and yet here we
+have two components maintaining their own state. That's a lot of state in such a
+small app!
+
+## How to minimize state with `props`
+
+So what can we do to minimize state? Well, let's think. Really, our two components
+are both lists; they aren't really all that different. What if we just
+conceive of our two components as one simple "presentational" or "dumb" component
+called `FruitList`? This component will just know how to render a list of fruits.
+And where will it get this list? That's right...it will receive the list as a
+`prop`. So our new `FruitList` component can even be implemented as a stateless
+functional component:
+
+```javascript
+const FruitList = (props) => {
+  return (
+    <ul>
+      {props.fruitList.map((fruit) => {
+        return <li key={fruit}>{fruit}</li>;
+      })}
+    </ul>
+  );
+}
+```
+
+Beautiful, right?! So much simpler and more stable. But what about our state? How
+will this newly simplified `FruitList` know which fruits to display? Well, all that
+can now be handled at the top of our simple application, by the `FruitList`'s
+parent. Let's say, to keep our example very simple, `FruitList` is embedded in a
+top-level component `App`. `App` could be implemented something like this:
+
+```javascript
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showEaten: false,
+      fruits: []
+    };
+
+    this.updateFruitList = this.updateFruitList.bind(this);
+    this.toggleListMode = this.toggleListMode.bind(this);
+  }
+
+  componentWillMount() {
+    this.updateFruitList();
+  }
+
+  toggleListMode() {
+    this.set({ showEaten: !this.state.showEaten });
+  }
+
+  updateFruitList() {
+    fetch('/api/fruit?eaten=' + this.state.showEaten).then((data) => {
+      this.setState({ fruits: data });
+    });
+  }
+
+  render() {
+    render <FruitList fruitList={this.state.fruits} />;
+  }
+}
+```
+So what have we done here? Well, there's a bunch of code here, but really all
```

Comment:
I think the word is appropriate here. It minimizes the significance of the changes, which is what I wanted to achieve.